pkg/nfd-master/nfd-master.go

```diff
@@ -141,13 +142,13 @@ func NewNfdMaster(args Args) (NfdMaster, error) {
 	// Initialize Kubernetes API helpers
 	nfd.apihelper = apihelper.K8sHelpers{Kubeconfig: args.Kubeconfig}
 
-	clientConfig, err := rest.InClusterConfig()
+	restConfig, err := rest.InClusterConfig()
```

Comment:
Import for rest ("k8s.io/client-go/rest") has been removed above

Reply:
it was moved above, I updated all imports